packages/babel-traverse/src/path/evaluation.ts

```diff
@@ -264,9 +264,13 @@ function _evaluate(path: NodePath, state: State): any {
     if (resolved === path) {
       deopt(path, state);
       return;
-    } else {
-      return evaluateCached(resolved, state);
     }
+    const value = evaluateCached(resolved, state);
+    if (typeof value === "object" && value !== null && binding.references > 1) {
```

Comment:
1 reference is fine because it would be the one we call `.evaluate()` on, right?

Reply:
Yes!